bridges/bin/runtime-common/src/extensions/refund_relayer_extension.rs

```diff
@@ -66,7 +65,7 @@ type CallOf<R> = <R as frame_system::Config>::RuntimeCall;
 /// coming from this parachain.
 pub trait RefundableParachainId {
 	/// The instance of the bridge parachains pallet.
-	type Instance;
+	type Instance: 'static;
```

Comment:
In the original PR, this trait and its implementation were moved to the `lib.rs` of this crate (https://github.com/paritytech/parity-bridges-common/pull/2873/files#diff-ec715be7be40d34a338721f0743eaeecf138cd3b007fb75a4a622ec605c63b3cR41), but it makes no sense - I've reverted this move